zitadel/provider.go

```diff
@@ -238,6 +262,8 @@ func providerConfigure(ctx context.Context, d *schema.ResourceData) (interface{}
 	clientinfo, err := helper.GetClientInfo(
 		d.Get(helper.InsecureVar).(bool),
 		d.Get(helper.DomainVar).(string),
+		d.Get(helper.JWTProfileFile).(string),
```

Comment:
As "token" was required, now all three attributes are optional, with no check that at least one is set.
The client will return an error, but would be easier to check it here right?

Reply:
Okay I'll add the validation for it in the function `helper.GetClientInfo`